src/main/java/org/terasology/launcher/util/BundleUtils.java

```diff
@@ -111,6 +116,26 @@ public static URL getFXMLUrl(String key, String relative) {
     }
 
     public static String getStylesheet(String key) {
```

Comment:
While you're at it: let's add some Javadoc here to state that this returns the path/URI(?) to the stylesheet.

Reply:
It's return valid URL notation for java(with them protocol handler)
Ok, some java docs needs :3